packages/spec-parser/tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
   "compileOnSave": false,
   "compilerOptions": {
-    "module": "ES2015",
+    "module": "ES6",
     "outDir": "dist",
     "declarationDir": "dist",
-    "target": "es2016",
+    "target": "ES6",
```

Comment:
Do we need to change it back?

Reply:
I will check and change back if necessary in next pr.